Makefile

```diff
@@ -2,9 +2,15 @@
 django-tests:
 	python manage.py test
 
-performance-tests:
+load-tests:
 	locust -f tests/load/smoke.py --host=https://developer.allizom.org
 
 # Note: this target should be run from the host machine with selenium running
 browser-tests:
 	pushd tests/ui ; ./node_modules/.bin/intern-runner config=intern-local d=developer.allizom.org b=firefox; popd
+
+clean:
+	find kuma -name '*.pyc' -exec rm {} \;
```

Comment:
:+1: for a ``make clean``.  I'll add the "really clean *.pyc except for that one python-magic file" later.